### DISCUSSION TITLE
Had to reinstall Plausible, no problems but any benefits from maxmind over DB-IP

### DISCUSSION THREAD

**Discussion by riekusr** [plausible/analytics#986] [Self-Hosted Support] [answered]
I remember it was a bitch to setup, and just noticed it (now) comes with DB-IP. Is there any downside as to leaving it like this default?

My vps is running on its limits so leaving out a container would help. 

**Thank you for your beautiful product.**

**Comment by ukutaht** [comment, marked as answer]
I don't know the difference in accuracy between DB-IP vs MaxMind. If you want to free up resources I'd say remove the MaxMind container and just keep DB-IP.
